src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets

```diff
@@ -660,8 +658,7 @@
   </Target>
 
   <Target Name="EnsureEmptyPackage"
-          DependsOnTargets="DetermineRuntimeDependencies"
-          BeforeTargets="GenerateNuSpec">
```

Comment:
Simpler mental model to be consistent in one set of targets.
BeforeTargets / AfterTargets is fine, it's just complicated when you're using a mix of both.  Sometimes they are necessary when you can't add the right DependsOnTargets, EG: if the target you are sequencing with is in somefile you don't own and that file didn't have an extensible list.